jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -586,26 +585,21 @@
           (unless av
             (ros::ros-error "[:pick-object-with-movable-region] arm:~a IK before trying grasp fails. Abort picking" arm)
             (return-from :pick-object-with-movable-region :ik-failed)))
-        ;; Move whole arm to target pose
+        ;; Move whole arm to location above of the target pose
         (send *ri* :angle-vector-sequence-raw next-avs
               :fast (send *ri* :get-arm-controller arm :head t) 0 :scale 5.0)
         (send *ri* :wait-interpolation))
+      ;; Now hand is inside of the bin, so we can use cylindrical
+      (send *ri* :move-hand arm (send *baxter* :hand-grasp-pre-pose arm :cylindrical) 1000)
```

Comment:
Also, I have a question: why did you use `:cylindrical`?
`:spherical` is currently used in suction because finger can run away from two direction force (difficult to explain...).